templates/privacy-preferences-modal.php

```diff
@@ -70,7 +70,21 @@
 								<?php foreach ( $args['consent_types'] as $consent_key => $consent_type ) : ?>
 									<div class="gdpr-cookies-used">
 										<div class="gdpr-cookie-title">
-											<p><?php echo esc_html( $consent_type['name'] ); ?></p>
+											<p>
```

Comment:
I think the `<p>` tag should be inside the loop below.

So it's something like:

```
if statement true:
    <a href="#">stuff</a>
else :
    <p>stuff</p>
```

Reply:
How come? The suggestion will create double `<p>` tags, only the link + name or name gets printed inside the `<p>` tags

Reply:
It'll only make double `<p>` tags if you don't remove the `<p>` tags outside the loop, right?

Do you need to do `<p><a>TEXT</a></p>`? Won't just `<a>TEXT</a>` do the same job?

Reply:
Just tested it, there's some styling in the plugin that targets those `<p>` tags which we could move it to the custom css but that seems a bit pointless as it's already in the plugin css.

Reply:
Fair enough